_includes/footer.html

```diff
@@ -48,8 +49,11 @@ <h4>&nbsp;</h4>
     <div class="column d-6-12 text-right">
       <ul class="social-media-links">
         {% for account in site.social %}
-        <li><a href="{{ account.link }}"><i class="fa fa-{{ account.icon }}" aria-hidden="true"
-              title="{{ account.title }}"></i></a></li>
+        <li>
+          <a aria-label="{{ account.title }}" href="{{ account.link }}">
+            <i class="fa-brands fa-{{ account.icon }}" aria-hidden="true" title="{{ account.title }}"></i>
```

Comment:
The `fa-brands` change here was to allow us to upgrade to FontAwesome 6 so we could add TikTok to the social links. Do we change back to `fa` here or also upgrade to FontAwesome 6 on the docs site + add TikTok to socials as part of this PR?

Reply:
For reference https://github.com/srobo/website/pull/434/files

Reply:
Ah, right. I'll change this back to `fa` and we can handle that separately unless you think it would be easy to roll in?

Reply:
Think the upgrade might be worth its own PR. Changes may required on other pages/parts of the site

Reply:
#357 handles the font-awesome update.